.github/workflows/gerrit-compose-required-rtdv3-merge.yaml

```diff
@@ -79,6 +79,8 @@ jobs:
         id: setup-python
         with:
           python-version: "3.8"
+      - name: Install graphviz
+        uses: tlylt/install-graphviz@main
```

Comment:
Please do not use `@main` but either a tag or better, the SHA related to the tag

Reply:
updated